src/features/click-to-play.js

```diff
@@ -1654,20 +1673,90 @@ function copyStylesTo (originalStyles, element) {
     element.style.cssText += cssText
 }
 
+/**
+ * Create a `<style/>` element with DDG font-face styles/CSS
+ * to be attached to DDG wrapper elements
+ * @returns HTMLStyleElement
+ */
+function makeFontFaceStyleElement () {
+    // Put our custom font-faces inside the wrapper element, since
+    // @font-face does not work inside a shadowRoot.
+    // See https://github.com/mdn/interactive-examples/issues/887.
+    const fontFaceStyleElement = document.createElement('style')
+    fontFaceStyleElement.textContent = styles.fontStyle
+    return fontFaceStyleElement
+}
+
+/**
+ * Create a `<style/>` element with base styles for DDG social container and
+ * button to be attached to DDG wrapper elements/shadowRoot, also returns a wrapper
+ * class name for Social Container link styles
+ * @param {"lightMode" | "darkMode"} mode Light or Dark mode value
+ * @returns {{wrapperClass: string, styleElement: HTMLStyleElement; }}
+ */
+function makeBaseStyleElement (mode = 'lightMode') {
+    // Style element includes our font & overwrites page styles
+    const styleElement = document.createElement('style')
+    const wrapperClass = 'DuckDuckGoSocialContainer'
+    styleElement.textContent = `
+        .${wrapperClass} a {
+            ${styles[mode].linkFont}
+            font-weight: bold;
+        }
+        .${wrapperClass} a:hover {
+            ${styles[mode].linkFont}
+            font-weight: bold;
+        }
+        .DuckDuckGoButton {
+            ${styles.button}
+        }
+        .DuckDuckGoButton > div {
+            ${styles.buttonTextContainer}
+        }
+        .DuckDuckGoButton.primary {
+           ${styles[mode].buttonBackground}
+        }
+        .DuckDuckGoButton.primary > div {
+           ${styles[mode].buttonFont}
+        }
+        .DuckDuckGoButton.primary:hover {
+           ${styles[mode].buttonBackgroundHover}
+        }
+        .DuckDuckGoButton.primary:active {
+           ${styles[mode].buttonBackgroundPress}
+        }
+        .DuckDuckGoButton.secondary {
+           ${styles.cancelMode.buttonBackground}
+        }
+        .DuckDuckGoButton.secondary > div {
+            ${styles.cancelMode.buttonFont}
+         }
+        .DuckDuckGoButton.secondary:hover {
+           ${styles.cancelMode.buttonBackgroundHover}
+        }
+        .DuckDuckGoButton.secondary:active {
+           ${styles.cancelMode.buttonBackgroundPress}
+        }
+    `
+    return { wrapperClass, styleElement }
+}
+
 function makeTextButton (linkText, mode) {
     const linkElement = document.createElement('a')
     linkElement.style.cssText = styles.headerLink + styles[mode].linkFont
     linkElement.textContent = linkText
     return linkElement
 }
 
-function makeButton (buttonText, mode) {
+function makeButton (buttonText, mode = 'lightMode') {
```

Comment:
Any chance you could add a JSDoc comment for this function that explains the idea of the `mode` (possible values and what they do / what they are for)?